model_servers/llamacpp_python/Makefile

```diff
@@ -12,10 +26,23 @@ mistral-7b-instruct-v0.1.Q4_K_M.gguf:
 install:
 	pip install -r tests/requirements.txt
 
-.PHONY: run
-run:
-	podman run -it -d -p 8001:8001 -v ./models:/locallm/models:ro,Z -e MODEL_PATH=models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=8001 --net=host ghcr.io/redhat-et/model_servers
+.PHONY: run-linux
+run-linux:
+	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models:ro,Z -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=$(PORT) --net=host $(IMAGE); cd model_servers/llamacpp_python;
+
+.PHONY: run-darwin
+run-darwin:
+	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=$(PORT) --net=host $(IMAGE); cd model_servers/llamacpp_python;
+
+.PHONY: run-linux-local
+run-linux-local:
+	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models:ro,Z -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=$(PORT) $(IMAGE); cd model_servers/llamacpp_python;	
+
+.PHONY:  run-darwin-local
+run-darwin-local:
+	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=$(PORT)  $(IMAGE); cd model_servers/llamacpp_python;
+
```

Comment:
what's the difference when running local (I'm staring but I'm not seeing it :) ) ? Also, for _all_ 3, you can add the `:ro` - it's the `Z` that errors when running in MacOS

Reply:
UNAME := $(shell uname)

RELABLE=""
ifeq ($(UNAME), Linux)
RELABEL=":Z"
endif
	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf:ro$(RELABEL)  -e HOST=0.0.0.0 -e PORT=$(PORT)  $(IMAGE); cd model_servers/llamacpp_python;